docs/custom.css

```
@@ -1196,7 +1196,7 @@ article p {
     }
   }
 
-  li[data-contextual-sidebar='true'] > .menu__list-item-collapsible > .menu__link::after {
+  li[data-contextual-sidebar='true'] > .menu__link::after {
```

Stylelint (17.14.0)

[error] Expected class selector ".menu__link" to be kebab-case (selector-class-pattern)

(selector-class-pattern)